index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@Sir_Malcom</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <img class="logo" src="./elements/lg.png" alt="Logo">
```

Comment:
In the design, this element is a text. Kindly adjust your design to meet this requirement. 
![image](https://user-images.githubusercontent.com/12239821/170706164-ba5acdfe-7d26-48f6-bf4b-a98e2d580415.png)

Reply:
Also, this should be a link to the homepage so, use a `<a>` tag to wrap the text.